db/migrate/20251211000000_add_reported_leaving_by_school_to_periods.rb

```diff
@@ -0,0 +1,9 @@
+class AddReportedLeavingBySchoolToPeriods < ActiveRecord::Migration[7.1]
+  def change
+    add_column :ect_at_school_periods, :reported_leaving_by_school_id, :bigint
+    add_index :ect_at_school_periods, :reported_leaving_by_school_id
+
+    add_column :mentor_at_school_periods, :reported_leaving_by_school_id, :bigint
+    add_index :mentor_at_school_periods, :reported_leaving_by_school_id
+  end
```

Comment:
Do we need to add a foreign key on `:reported_leaving_by_school_id`? - I'm thinking it might protect against invalid or orphaned school IDs